checkbox-ng/plainbox/test_provider_manager.py

```diff
@@ -312,11 +323,22 @@ def test_develop__force(self, mock_path_entry):
         self.tool.main(["develop", "--force"])
         self.assertFileContent(filename, content)
 
+        # PROVIDERPATH defined
+        mock_getenv.return_value = "/tmp/com.example.test.provider"
+        filename = "/tmp/com.example.test.provider/com.example.test.provider"
+        os.makedirs(os.path.dirname(filename), exist_ok=True)
+        with open(filename, "wt+") as stream:
+            stream.write("should have been overwritten")
+        self.tool.main(["develop", "--force"])
+        self.assertFileContent(filename, content)
```

Comment:
Same for this test

Reply:
This is ok, note, filename is overwritten here so its the correct name